src/Symfony/Component/VarDumper/Server/DumpServer.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Server;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\VarDumper\Cloner\Data;
+
+/**
+ * A server collecting Data clones sent by a ServerDumper.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ *
+ * @final
+ */
+class DumpServer
+{
+    public const HOST_ENV_VAR = 'VAR_DUMPER_SERVER';
+    public const DEFAULT_HOST = 'tcp://0.0.0.0:9912';
+
+    private $host;
+    private $socket;
+    private $logger;
+
+    /**
+     * @param string|null          $host   The server host or null to read it from the VAR_DUMPER_SERVER env var
+     * @param LoggerInterface|null $logger
```

Comment:
I think the docblock can be removed altogether

Reply:
The docblock above still is useful to me to highlight the `VAR_DUMPER_SERVER` env var, but the $logger one could be removed. However, we didn't agree on partial docblock in the core, right?